LambdaEngine/Source/Game/ECS/Systems/Networking/ClientSystem.cpp

```diff
@@ -312,12 +311,10 @@ namespace LambdaEngine
 		}
 	}
 
-	bool ClientSystem::CompareGameStates(const GameState& gameStateLocal, const GameState& gameStateServer)
+	bool ClientSystem::CompareGameStates(const GameState& gameStateServer, const GameState& gameStateLocal)
```

Comment:
Local first then Server
```suggestion
	bool ClientSystem::CompareGameStates(const GameState& gameStateLocal, const GameState& gameStateServer)
```